tests/data_files/Readme-x509.txt

```diff
@@ -91,3 +95,13 @@ Signing CA in parentheses (same meaning as certificates).
 
 Note: crl_future would revoke server9 and cert_sha384.crt if signed by CA 1
       crl-rsa-pss* would revoke server6.crt if signed by CA 2
+
```

Comment:
You are adding to this folder scripts and Makefiles and openssl config files. Could you please also document the rationale for having these around?